docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -11,447 +11,183 @@ menu:
 type: docs
 ---
 
-YugabyteDB Anywhere can check universes for issues that may affect deployment. Should problems arise, YugabyteDB Anywhere can automatically issue alert notifications
+YugabyteDB Anywhere has default, preconfigured alerts, both at the YugabyteDB Anywhere and universe level. Universe alerts can be configured globally for all universes, or per specific universe. In addition to the default alerts, you can configure your own alerts based on a specific condition on any available metric.
 
-For additional information, see the following:
+To review alerts that have triggered, refer to [Alerts](../../alerts-monitoring/alert/). To view performance metrics, refer to [Use metrics](../../troubleshoot/universe-issues/#use-metrics/).
 
-- [Alerts](../../alerts-monitoring/alert/)
-- [Metrics](../../troubleshoot/universe-issues/#use-metrics/)
-- [Alerts and Notifications in YugabyteDB Anywhere](https://www.yugabyte.com/blog/yugabytedb-2-8-alerts-and-notifications/)
-
-You can use preconfigured alerts provided by YugabyteDB Anywhere, or create and configure your own alerts based on the metrics' conditions.
-
-You can access YugabyteDB Anywhere health monitor and configure alerts by navigating to **Admin > Alert Configurations**, as per the following illustration:
-
-![Configure alerts](/images/yp/config-alerts1.png)
+You can access YugabyteDB Anywhere health monitor and configure alerts by navigating to **Admin > Alert Configurations**.
```

Comment:
To me, health monitor is confusing because there is a separate health tab on **Admin > Alert Configurations > Health**.  So when you actually get to Admin > Alert Configurations , You see "Alert policies" as the default selection and maybe you can rephrase it to "You can access YugabyteDB Anywhere access policies and configure alerts by navigating to **Admin > Alert Configurations** "